src/main/java/stream/flarebot/flarebot/commands/secret/GuildCommand.java

```diff
@@ -0,0 +1,114 @@
+package stream.flarebot.flarebot.commands.secret;
+
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.entities.*;
+import stream.flarebot.flarebot.FlareBot;
+import stream.flarebot.flarebot.commands.Command;
+import stream.flarebot.flarebot.commands.CommandType;
+import stream.flarebot.flarebot.FlareBotManager;
+import stream.flarebot.flarebot.objects.GuildWrapper;
+import stream.flarebot.flarebot.util.MessageUtils;
+
+import java.awt.*;
```

Comment:
Go to import settings in codes style and remove the section for java awt all together.